.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
+.apitester.js
```

Comment:
I think you need to add node_modules to your gitignore, it says you've changed 5000+ files and the page is impossibly laggy haha